x-pack/plugins/enterprise_search/public/applications/app_search/icons/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { CursorIcon } from './cursor_icon';
+export { EngineIcon } from './engine_icon';
+export { MetaEngineIcon } from './meta_engine_icon';
```

Comment:
Can you update `icons.test.tsx` to cover the newly added CursorIcon file as well? The final file should probably look like this:

```js

import { CursorIcon } from './cursor_icon';
import { EngineIcon } from './engine_icon';
import { MetaEngineIcon } from './meta_engine_icon';

describe('Shared icons', () => {
  it('renders a cursor icon', () => {
    const wrapper = shallow(<CursorIcon />);
    expect(wrapper.hasClass('euiIcon')).toBe(true);
  });

  it('renders an engine icon', () => {
    const wrapper = shallow(<EngineIcon />);
    expect(wrapper.hasClass('engineIcon')).toBe(true);
  });

  it('renders a meta engine icon', () => {
    const wrapper = shallow(<MetaEngineIcon />);
    expect(wrapper.hasClass('engineIcon')).toBe(true);
  });
});
```

Reply:
https://github.com/elastic/kibana/pull/92939/commits/f9fa1ace4cda0fe3e5b669de883ecaadcbc6e3aa